packages/loot-core/src/server/api.ts

```diff
@@ -226,6 +229,18 @@ handlers['api/download-budget'] = async function ({ syncId, password }) {
   await handlers['load-budget']({ id: result.id });
 };
 
+handlers['api/get-budgets'] = async function () {
+  const budgets: (Budget | APIRemoteFileEntity)[] =
+    await handlers['get-budgets']();
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I don't think the current type is accurate here
```suggestion
  const budgets: Budget[] =
    await handlers['get-budgets']();
```

Reply:
Because of the concat below, this was the type needed.  If I change to a destructure instead of concat, then the type could be changed here.